### ISSUE TITLE
FutureWarning line_terminator

### ISSUE THREAD

**Issue by cmosguy** [yehoshuadimarsky/bcpandas#87]
hey @yehoshuadimarsky -

This is a super awesome package and has totally saved me a ton of time and headaches, well done to everyone who contributed!

Just a heads up, there is a `FutureWarning`:

```
/slowfs/us01dwt2p298/NVM/designs/kleinada/ds_work/miniconda3/envs/nvm/lib/python3.10/site-packages/bcpandas/main.py:399: FutureWarning: the 'line_terminator'' keyword is deprecated, use 'lineterminator' instead.
  df.replace({True: 1, False: 0}).to_csv(
```

Someone may want to fix this.

**Comment by jasonni0** [comment]
Looks like this already solved

**Comment by borchero** [comment]
Yes, this has been solved in v2.4.0 :)
